modules/installation-configuration-parameters.adoc

```diff
@@ -1094,6 +1114,36 @@ This example defines no affinity groups:
       affinityGroupNames: []
 ----
 |String
+|`<machine-pool>.platform.ovirt.AutoPinningPolicy`
+| Optional. AutoPinningPolicy defines the policy to automatically set the CPU and NUMA settings including pinning to the host for the instance. When the field is omitted, the default is `none`. Supported values: `none`, `resize_and_pin`.
```

Comment:
Maybe give a link to the RHV docs about Auto Pinning Policy? not a must, only if you think this is needed